src/App.snapshot.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
There is an issue with test, but it's out of scope for current task: you have forgotten to mock your css files, but jest tries to parse them as they were .js